--- hugs98-Nov2003/docs/users_guide/Makefile.in~	Fri Feb 13 16:30:38 2004
+++ hugs98-Nov2003/docs/users_guide/Makefile.in	Fri Feb 13 17:08:51 2004
@@ -4,9 +4,9 @@
 DOCBOOK2PS   = docbook2ps
 
 ROOT	     = users_guide
-PARTS	     = $(ROOT).sgml haskell98.sgml hugs_ghc.sgml hugs_only.sgml \
-	       introduction.sgml license.sgml miscellaneous.sgml \
-	       options.sgml using_hugs.sgml
+PARTS	     = $(ROOT).xml haskell98.xml hugs_ghc.xml hugs_only.xml \
+	       introduction.xml license.xml miscellaneous.xml \
+	       options.xml using_hugs.xml
 # Options needed for non-SuSE $(DOCBOOK2HTML)
 HTML_OPTS    = --output $(ROOT) -V '%use-id-as-filename%'
 
@@ -40,16 +40,16 @@
 	rm -rf $(ROOT)
 	test -z '$(DOCBOOK2HTML)' || \
 	  if $(DOCBOOK2HTML) --help | grep -e --output > /dev/null 2>&1 ; then \
-	    $(DOCBOOK2HTML) $(HTML_OPTS) $(ROOT).sgml ; \
+	    $(DOCBOOK2HTML) $(HTML_OPTS) $(ROOT).xml ; \
 	  else \
-	    $(DOCBOOK2HTML) $(ROOT).sgml ; \
+	    $(DOCBOOK2HTML) $(ROOT).xml ; \
 	  fi
 
 $(ROOT).dvi: $(PARTS)
-	test -z '$(DOCBOOK2DVI)' || $(DOCBOOK2DVI) $(ROOT).sgml
+	test -z '$(DOCBOOK2DVI)' || $(DOCBOOK2DVI) $(ROOT).xml
 
 $(ROOT).ps: $(PARTS)
-	test -z '$(DOCBOOK2PS)' || $(DOCBOOK2PS) $(ROOT).sgml
+	test -z '$(DOCBOOK2PS)' || $(DOCBOOK2PS) $(ROOT).xml
 
 $(ROOT).pdf: $(PARTS)
-	test -z '$(DOCBOOK2PDF)' || $(DOCBOOK2PDF) $(ROOT).sgml
+	test -z '$(DOCBOOK2PDF)' || $(DOCBOOK2PDF) $(ROOT).xml
